.pylintrc

```diff
@@ -0,0 +1,44 @@
+[MASTER]
+
+# Add files or directories to the blacklist.
+ignore=LICENSE,
+       MANIFEST.in,
+       PUBLISH.md,
+       README.md,
+       requirements.txt,
+       script.test,
+
+
+[MESSAGES CONTROL]
+
+disable=broad-except,
+        too-few-public-methods,
+        invalid-name,
+        attribute-defined-outside-init,
+        useless-object-inheritance,
+        no-else-return,
+        no-name-in-module,
```

Comment:
I do not have to disable this check.

Reply:
It is a problem with pylint when using virtual env. That's why it was showing this error in my pylint check. I am removing this for now from disabled list.
https://github.com/PyCQA/pylint/issues/1524

Reply:
Although I don't like to disable `no-name-in-module` and `import-error`, I see the reason now in travis log.
Please readd it.